src/components/20-molecules/accordeon/index.js

```diff
@@ -0,0 +1,123 @@
+/* eslint-disable import/no-extraneous-dependencies */
+import { LitElement, html, css, unsafeCSS, svg } from 'lit-element';
+import expandLess from '@axa-ch/materials/icons/material-design/expand_less.svg';
+import expandMore from '@axa-ch/materials/icons/material-design/expand_more.svg';
+
+import { classMap } from 'lit-html/directives/class-map';
+import { defineVersioned } from '../../../utils/component-versioning';
+import { applyDefaults } from '../../../utils/with-react';
+import styles from './index.scss';
+
+class AXAAccordeon extends LitElement {
+  static get tagName() {
+    return 'axa-accordeon';
+  }
+
+  static get styles() {
+    return css`
+      ${unsafeCSS(styles)}
+    `;
+  }
+
+  static get properties() {
+    return {
+      small: { type: Boolean },
+      open: { type: Boolean },
+      disabled: { type: Boolean },
+      icon: { type: String },
+      title: { type: String },
+    };
+  }
+
+  constructor() {
+    super();
+    applyDefaults(this);
+  }
+
+  toggleAccordion() {
+    if (!this.disabled) {
+      this.open = !this.open;
+      this.toggleAnimation();
+    }
+  }
+
+  firstUpdated() {
+    this.getAllTagMatches().shadowRoot.querySelector(
```

Comment:
This method-name does not represent what is happening in there.

Maybe you can name the method "addBorderTopToFirstElement()" and do everything in there.